src/OpenDebugAD7/VariableManager.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Microsoft.DebugEngineHost.VSCode;
+using Microsoft.VisualStudio.Debugger.Interop;
+using Microsoft.VisualStudio.Shared.VSCodeDebugProtocol.Messages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenDebugAD7
+{
+    internal class VariableEvaluationData
+    {
+        internal IDebugProperty2 DebugProperty;
+        internal enum_DEBUGPROP_INFO_FLAGS propertyInfoFlags;
+    }
+
+    internal class VariableManager
+    {
+        private readonly HandleCollection<Object> m_variableHandles;
```

Comment:
>private [](start = 8, length = 7)

Suggestion: Add a comment:
// NOTE: The value being stored can be a IDebugStackFrame2 or a VariableEvaluationData